docs/docs/guides/limiting-data-using-filters.mdx

```diff
@@ -229,11 +229,24 @@ To add saved filters to your dashboard, you must be an `editor` or an `admin` fo
 
 ### Some things worth mentioning about dashboard filters
 
-#### Users can remove and override _saved filters_ on a dashboard while they're viewing it
+#### Sharing your dashboard with overrides
 
-Users have the option to delete or edit a dashboard's saved filters while they're looking at it. But, unless they're making these changes in `edit` mode and saving them, the original saved filters will be applied for the next user that comes to look at the dashboard.
```

Comment:
removed this part because it can be confusing. Although we can remove a saved filter, click apply, and view how it impacts the dashboard, I don't think it's worth mentioning